src/app/albumList/albumList.component.ts

```diff
@@ -59,6 +71,16 @@ export class AlbumListPage {
     this.menu.enable(false);
   }
 
+  showDetails(album:Album) {
+    this.navCtrl.push(StoryListPage, {
+      "album": album,
+    });
+  }
+
+  setBackground(story:UserStory){
```

Comment:
I will rename to getBackground because we are getting the background thanks to the API